### PR TITLE
Feature/volume status btrfs for CC-1171

### PR DESCRIPTION
**Pull request by morr-zenoss**

### CODE REVIEW

File: cli/cmd/volume.go [hunk 1]
```diff
@@ -31,11 +31,11 @@ func (c *ServicedCli) initVolume() {
 		Description: "",
 		Subcommands: []cli.Command{
 			{
-				Name:         "status",
-				Usage:        "Provides volume status for specified tenants",
-				Description:  "serviced volume status [TENANT [TENANT ...]]",
-				Action:       c.cmdVolumeStatus,
-				Flags:  []cli.Flag{
+				Name:        "status",
+				Usage:       "Provides volume status for specified tenants",
```
Comment (iancmcc) [inline cli/cmd/volume.go]: No longer for specified tenants.

File: rpc/master/volume_client.go [hunk 2]
```diff
@@ -12,16 +12,16 @@
...
 	"github.com/zenoss/glog"
 )
 
 //GetVolumeStatus gets status information for the given volume or nil
-func (c *Client) GetVolumeStatus(volumeNames []string) (*volume.Statuses, error) {
-	glog.V(2).Infof("[hosts_client.go]master.GetVolumeStatus(%v)", volumeNames)
+func (c *Client) GetVolumeStatus() (*volume.Statuses, error) {
 	response := &volume.Statuses{}
-	if err := c.call("GetVolumeStatus", volumeNames, response); err != nil {
+	if err := c.call("GetVolumeStatus", empty, response); err != nil {
```
Comment (iancmcc) [inline rpc/master/volume_client.go]: Where does empty come from?

Comment (morr-zenoss) [inline rpc/master/volume_client.go]: it's defined in client.go

Comment (iancmcc) [inline rpc/master/volume_client.go]: Cool

**Pull request merged**